stable/xray/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: xray
 home: https://www.jfrog.com/xray/
-version: 3.3.0
-appVersion: 3.3.0
+version: 3.3.1
+appVersion: 3.3.1
```

Comment:
@vinayagg Can you revert bumping appVerison ?

Reply:
Fixed. Thank you for watching out.